handler/src/main/java/io/netty/handler/ssl/ReferenceCountedOpenSslEngine.java

```diff
@@ -881,98 +915,74 @@ public final SSLEngineResult unwrap(
                         srcsOffset++;
                         continue;
                     }
+
                     // Write more encrypted data into the BIO. Ensure we only read one packet at a time as
                     // stated in the SSLEngine javadocs.
-                    int written = writeEncryptedData(src, min(packetLengthRemaining, src.remaining()));
+                    int written = writeEncryptedData(src, Math.min(packetLength, remaining));
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Unnecessary use of fully qualified name 'Math.min' due to existing static import 'java.lang.Math.min' [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=pmd%3AUnnecessaryFullyQualifiedName)